src/assets/server/rest/v1/docs/e-mobility-oas.json

```diff
@@ -2645,7 +2679,7 @@
             "$ref": "#/components/parameters/SiteID"
           },
           {
-            "$ref": "#/components/parameters/SiteAreaID"
+            "$ref": "#/components/parameters/SiteAreaIDs"
```

Comment:
SiteAreaID should always be singular like any other query params where we can pass several values that are separated by a pipe.

Reply:
Actually, the Query param's name itself is not modified. I just changed the name of the param in the oas file to differentiate Single ID / Multiple IDs.
As the TriggerSmartCharging endpoint focus on one SiteAreaID, we should not use a param labeled as "Pipe separated IDs"

Reply:
> Actually, the Query param's name itself is not modified. I just changed the name of the param in the oas file to differentiate Single ID / Multiple IDs.
> As the TriggerSmartCharging endpoint focus on one SiteAreaID, we should not use a param labeled as "Pipe separated IDs"

Okay got it, then we should also align the naming of all the other query params ;-)

Reply:
Indeed! Done